internal/pkg/cli/project_delete.go

```diff
@@ -268,6 +272,21 @@ func (o *deleteProjOpts) deleteLocalWorkspace() error {
 	return nil
 }
 
+func (o *deleteProjOpts) deleteProjectPipeline() error {
+	vars := deletePipelineVars{
+		GlobalOpts:       NewGlobalOpts(),
+		SkipConfirmation: true,
+		DeleteSecret:     true,
+	}
+
+	deletePipelineOpts, err := newDeletePipelineOpts(vars)
+	if err != nil {
+		return err
+	}
+
+	return deletePipelineOpts.Run()
```

Comment:
since we construct the `vars` ourselves we prob don't need the `Run()` method and call `Execute()` directly

Reply:
True, but it makes the error handling a little less verbose, plus it still does the validation that helps with debugging. The "Ask" is still in there but it'll get skipped since we'll always skipConfirmation.